pg_diffix--0.0.1.sql

```diff
@@ -62,6 +63,15 @@ AS $$
   END;
 $$ LANGUAGE plpgsql;
 
+CREATE PROCEDURE unmark_table(table_name text)
+AS $$
+  BEGIN
+    DELETE FROM pg_catalog.pg_seclabel WHERE provider = 'pg_diffix' AND objoid = table_name::regclass::oid AND label = 'aid';
```

Comment:
Can't we do the same for table?

Reply:
I don't know what you mean? These lines are done for a table

Reply:
For table there is `SECURITY LABEL FOR pg_diffix ON TABLE ' || table_name || ' IS NULL`.
I think the same `DELETE` can clean up both table and AID cols.

Reply:
Ah, I see, remove the `label = 'aid'` and then we don't need the setting to NULL. Yep this works, isn't this a bit too "magic" though? And not 100% sure if there cannot be something else sitting with the same `objoid`, other than columns :thinking:

Reply:
but, on second thought, we unmark the table, so whatever is tagged onto it and concerning `pg_diffix` is fine to be unlabelled. I'll proceed like you suggest :+1:

Reply:
> but, on second thought, we unmark the table, so whatever is tagged onto it and concerning `pg_diffix` is fine to be unlabelled. I'll proceed like you suggest +1

Yes, this was my idea. Clear anything related to that table.